hc/settings.py

```diff
@@ -153,3 +161,7 @@
     from .local_settings import *
 else:
     warnings.warn("local_settings.py not found, using defaults")
+
+
+STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
+STATIC_URL = '/static/'
```

Comment:
I suppose [this](https://github.com/andela/healthchecks-thunknalla/pull/7#issuecomment-306603709) is why build [237540973](https://travis-ci.org/andela/healthchecks-thunknalla/builds/237540973) was failing.